.github/workflows/pypi.yaml

```diff
@@ -15,8 +15,8 @@ jobs:
 
     strategy:
       matrix:
-        os: [ 'macos-latest', 'ubuntu-20.04', 'windows-latest' ]
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        os: [ 'ubuntu-20.04' ]
+        python-version: [ '3.9' ]
```

Comment:
What is the reason for dropping these OSs and older versions? Not saying we shouldn't, just thinking of the rationale going forward.